build-tools/packages/build-tools/src/fluidBuild/tasks/leaf/tscTask.ts

```diff
@@ -485,6 +485,7 @@ export abstract class TscDependentTask extends LeafWithDoneFileTask {
 				const tsBuildInfo = await dep.readTsBuildInfo();
 				if (tsBuildInfo === undefined) {
 					// If any of the tsc task don't have build info, we can't track
+					this.traceError(`tsbuildinfo file not found for ${dep.name}: ${dep.command}`);
```

Comment:
`getDoneFileContent` is called for both before and after the task, right? Then of course in the clean build case there won't be one before - doesn't seem like an error.